products/sle15/profiles/pci-dss-4.profile

```diff
@@ -12,7 +12,7 @@ description: |-
     Ensures PCI-DSS v4 security configuration settings are applied.
 
 selections:
-    -  pcidss_3:all:base
+    -  pcidss_4:all:base
```

Comment:
I would recommend to separate the rule update and the PCI-DSS changes in two different PRs, if this was intentional here.

Reply:
Dropped it it was included by mistake here